### ISSUE TITLE
App not installed error when using app center distribution for android

### ISSUE THREAD

**Issue by sumanyadav12** [microsoft/appcenter#2293]
After build successfully I had distribute the android signed APK and download it and installed but it is showing me "App not installed." 
Please help me with any proper solution.
Thank you.

**Comment by AnastasiaKubova** [comment]
Hi, @sumanyadav12 ! Thanks for getting in touch with us! Could you please add some more details:
- Does this happen after downloading apk via the App Center Install portal or via the in-app update dialog in your app also?
- Does this issue happens on all Android devices or only on specific models?
- On what Android versions it happens?

**Comment by AnastasiaKubova** [comment]
Hi, @sumanyadav12 ! Thanks for getting in touch with us! Could you please add some more details:
- Does this happen after downloading apk via the App Center Install portal or via the in-app update dialog in your app also?
- Does this issue happens on all Android devices or only on specific models?
- On which versions of Android does this happen?

**Comment by bamboo-horobets** [comment]
Hello @AnastasiaKubova @DmitriyKirakosyan 

I'm facing the same issue now. Maybe these details will help to find a solution.
- It is happening during any available options to install (downloading .apk & using the App Center Install portal & via installation link & via scanning QR-code of specific build)
- This issue is reproduced on Android 11 and Android 12 versions. At the same time, the App installs on older Android devices without any error.
- This issue is reproducible on multiple Android devices (Google Pixel 4XL - Android 12 & Samsung A51 - android 11, Xiaomi)
- I've started facing this issue after I've updated the following lines inside "android/build.gradle" file. Here are the changes I've made:
```
buildscript {
    ext {
        buildToolsVersion = "29.0.3" => "30.0.2"
        minSdkVersion = 21
        compileSdkVersion = 29 => 30
        targetSdkVersion = 29 => 30
        ndkVersion = "20.1.5948944"
    }
```

Please help me to find the solution to this issue.
Thanks in advance.

![Screenshot_20211130-111048](https://user-images.githubusercontent.com/78470814/144078460-88cf8100-68ef-4a5b-bccb-ad165cd15c8f.png)
![Screenshot 2021-11-30 at 17 33 25](https://user-images.githubusercontent.com/78470814/144078488-6b8703f0-a75c-4b72-9d57-3942f0389cd5.png)

**Comment by AnastasiaKubova** [comment]
Hi, @bamboo-horobets ! Thanks for getting in touch with us!
Could you please share the [verbose logs](https://docs.microsoft.com/en-us/appcenter/sdk/other-apis/android#adjust-the-log-level)?

**Comment by bamboo-horobets** [comment]
Hello @AnastasiaKubova .

I was able to resolve this issue from my side.
The problem was here (see attached screenshots).
![Screenshot 2021-12-01 at 19 22 59](https://user-images.githubusercontent.com/78470814/144380252-ea57e414-e177-4c8e-8cd8-e84bc1330d53.png)
![Screenshot 2021-12-01 at 19 28 50](https://user-images.githubusercontent.com/78470814/144380260-45245e38-8ab1-40f0-bb9f-59c8763377d1.png)


I have reuploaded the keystone file and saved the configuration. This enabled the usage of the APK Signature Scheme v3 signing flow.
Therefore no help is required now :)

Thanks for your time and will to help.
